src/common/fs_manager.cpp

```diff
@@ -75,23 +89,40 @@ unsigned dir_node::remove(const gpid &pid)
     return iter->second.erase(pid);
 }
 
-void dir_node::update_disk_stat()
+void dir_node::update_disk_stat(bool &status_changed)
```

Comment:
```bool dir_node::update_disk_stat()```

Reply:
I used to clear function like your suggestion. If return false, it seems like update disk stat failed, so I pass `status_chaned` instead.